pkg/catalog/routes.go

```diff
@@ -333,12 +335,31 @@ func getTrafficPoliciesForService(mc *MeshCatalog, routePolicies map[trafficpoli
 		}
 	}
 
-	for _, trafficTarget := range srcDstTrafficTargetMap {
+	/*for _, trafficTarget := range srcDstTrafficTargetMap {
 		matchedTrafficTargets = append(matchedTrafficTargets, trafficTarget)
 	}
 
 	log.Debug().Msgf("Traffic policies for service %s: %+v", meshService, matchedTrafficTargets)
-	return matchedTrafficTargets, nil
+	return matchedTrafficTargets, nil*/
+
+	return srcDstTrafficTargetMap, nil
+}
+
+func (mc *MeshCatalog) getTrafficRoutesForService(svc service.MeshService, trafficTargets map[string]trafficpolicy.TrafficTarget) []trafficpolicy.TrafficRoutes {
```

Comment:
The arg for this function is what is returned by getTrafficPoliciesForService(). I will refactor a bit here after an initial review